cloud-formation/src/templates/lambda.yaml

```diff
@@ -0,0 +1,15 @@
+  {{name}}:
+    Type: "AWS::Lambda::Function"
+    Description: "{{description}}"
+    Properties:
+      {{> environmentVariables}}
+
+      FunctionName: !Sub ${Stack}-ContributionCompletedLambda-${Stage}
```

Comment:
Should the references to "CreatePaymentMethodLambda" not be `{{name}}` or similar?